Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -423,6 +423,8 @@ wxMenu* MainMenuBar::CreateSymbolsMenu() const
       IDM_SCAN_SIGNATURES, _("&Signature Database"),
       _("Recognise standard functions from Sys/totaldb.dsy, and use generic zz_ "
         "names for other functions."));
+  generate_symbols_menu->Append(IDM_SCAN_RSO, _("&RSO Modules"),
```

Comment:
Is it an issue since we have to enter the submenu in order to use RSO Modules' accelerator? It doesn't seem so on Windows at least.